.github/workflows/deploy-storybook.yml

```
@@ -0,0 +1,52 @@
+name: Deploy Storybook
+
+on:
+  pull_request:
+    types: [closed]
+    paths:
+      - 'client/**/*.stories.tsx'
+      - 'client/**/*.stories.ts'
+      - 'client/**/*.mdx'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy-storybook:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js and pnpm
+        uses: ./.github/actions/pnpm-setup-node
+
+      - name: Install dependencies
+        working-directory: ./client
+        run: pnpm install --frozen-lockfile
+
+      - name: Deploy with Chromatic
+        id: chromatic
+        uses: chromaui/action@v1
```

지금은 [최신 버전 13](https://github.com/chromaui/action)이라고 알고 있는데요! 다음과 같이 최신 버전 유지하는 것도 좋을 것 같아요~! 
```suggestion
        uses: chromaui/action@latest
```

---

오 이 부분은 미처 신경쓰지 못했네요! 
다른 리뷰까지 한꺼번에 반영했습니다 !

27fddc84a6eceed008aa7b72355380a3b36e784d